srcpkgs/kiten/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'kiten'
+pkgname=kiten
+version=20.04.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules pkg-config kcoreaddons kdoctools gettext-devel gettext"
+makedepends="qt5-devel qt5-declarative qt5-declarative-devel
+khtml-devel ki18n-devel knotifications-devel kdoctools-devel kxmlgui-devel karchive-devel kcoreaddons-devel"
+depends="khtml hicolor-icon-theme"
+short_desc="Japanese Reference/Study Tool"
+maintainer="basketdou <basketdou@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://kde.org/applications/education/org.kde.kiten"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=c5d002ba2ee4a11913457808da67f9c7ee5dbcf1731fad11842a4aa338862c82
+
+pre_check() {
+    export QT_QPA_PLATFORM=offscreen
+}
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+="kcompletion kcompletion-devel kconfig kconfig-devel kconfwidgets-devel
```

Comment:
Missing space before `kcompletion` corrupts `hostmakedepends`

Reply:
Fixed in the latest commit